### PR TITLE
Open ESD product tab testing

### PR DESCRIPTION
**Pull request by ermindedicss**
- Created the first script to open the ESD product tab
- Added some variables to the Cypress.json

### CODE REVIEW

File: src/Resources/app/administration/test/e2e/cypress/integration/esd_test_spec.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            }).then(() => {
+                cy.visit(`${Cypress.env('admin')}#/sw/product/index`);
+                cy.contains('Product name').click();
+            })
+    });
+    
+    it('Open ESD tab', function() {
+        const page = new ProductPageObject();
+
+        cy.get('.sw-tabs-item[title="ESD"]').click()
```
Comment (ChristopherDosin) [inline src/Resources/app/administration/test/e2e/cypress/integration/esd_test_spec.js:18]: @ermindedicss When you click the ESD tab - You're expecting something, no?
So to check if it's actually working - Like click the ESD Tab, then expect a button or what ever ( something that won't really change ).

https://docs.cypress.io/guides/references/assertions

Comment (ermindedicss) [reply src/Resources/app/administration/test/e2e/cypress/integration/esd_test_spec.js:18]: I just added an assertion for the tab content. There are an upload field and two checkbox fields in the ESD configuration. However, there are classes missing for them. Is this OK?

**Review by ChristopherDosin — APPROVED** [review]

File: src/Resources/app/administration/test/e2e/cypress/integration/esd_test_spec.js (hunk 1 above)
Comment (ChristopherDosin) [reply src/Resources/app/administration/test/e2e/cypress/integration/esd_test_spec.js:18]: If there are classes missing that you need - We need to add them.
We need to cover everything that a User is basically doing / clicking.

Comment (ermindedicss) [reply src/Resources/app/administration/test/e2e/cypress/integration/esd_test_spec.js:18]: This ticket was mostly to only open the tab. I can cover the rest right away if you want. 
And should I add the classes myself?

Comment (ChristopherDosin) [reply src/Resources/app/administration/test/e2e/cypress/integration/esd_test_spec.js:18]: Yes, but: Open the Tab -> Expecting the correct result. ( Not just click, we need to expect/check something to be sure you're on the correct page and it's loading )

If that Test is covered - Next Test case.

Comment (ermindedicss) [reply src/Resources/app/administration/test/e2e/cypress/integration/esd_test_spec.js:18]: In this case, the tab content has a unique class, `sas-product-detail-esd__generated-esd` so I think that should be it

**Pull request merged**